src/transformStateWithClones.js

```diff
@@ -7,7 +7,49 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const history = [];
+  let lastState = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
wrong syntaxes of `switch case`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch